src/home/homework-blob.c

```diff
@@ -0,0 +1,310 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later
+ *
+ * Copyright © 2024 GNOME Foundation Inc.
+ *      Original Author: Adrian Vovk
```

Comment:
Into its own comment.

Also need to fixup copyright comments elsewhere, where missing. Looks like I forgot to add one to `homework.c`, etc.